### PR TITLE
Fix for MacOS chrono cast in Atom_RHI.Metal.

### PR DESCRIPTION
**Pull request by lumberyard-employee-dm**
Signed-off-by: lumberyard-employee-dm <56135373+lumberyard-employee-dm@users.noreply.github.com>

relates #12176

### CODE REVIEW

**Review by moudgils — APPROVED** [review]

**Review by spham-amzn — APPROVED** [review]

File: Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ namespace AZ
                 releaseQueueDescriptor.m_collectLatency = m_descriptor.m_frameCountMax;
                 m_releaseQueue.Init(releaseQueueDescriptor);
             }
-
```
Comment (lawsonamzn) [inline Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp:62]: Unnecessary change?

**Review by lawsonamzn — APPROVED** [review]
Why so many unrelated whitespace changes?

**Review by amzn-phist — APPROVED** [review]

File: Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp (hunk 1 above)
Comment (lumberyard-employee-dm) [reply Gems/Atom/RHI/Metal/Code/Source/RHI/Device.cpp:62]: Ah, it is the trim trailing whitespace on save option in VSCode

**Comment by lumberyard-employee-dm** [comment]
Closing as PR #12176 has this change in it
